gnosis/safe-react
# feat: pending transactions widget

## What it solves
Resolves #3699

## How this PR fixes it
The pending transactions for the selected Safe are displayed in a Dashboard widget.

## How to test it
Open the route `/app/home`
If your first owned Safe in the LocalStorage has pending transactions, you will see them displayed in the "Transactions to Sign" card.
Click any of the transactions.
You will be redirected to the Safe's "Queued List"

## Analytics changes
None

## Screenshots
<img width="996" alt="Screen Shot 2022-04-05 at 08 19 50" src="https://user-images.githubusercontent.com/32431609/161692127-98ff4b57-3ffa-4471-a508-b032d97f0b36.png">
<img width="872" alt="Screen Shot 2022-04-06 at 09 32 45" src="https://user-images.githubusercontent.com/32431609/161920388-9a3d0b1d-d3a6-4ed5-ab2a-454c6360c97f.png">
<img width="871" alt="Screen Shot 2022-04-06 at 09 32 32" src="https://user-images.githubusercontent.com/32431609/161920415-e7b7e37b-bd21-4b23-8686-b2a9c3e8ec8b.png">
<img width="1537" alt="Screen Shot 2022-04-06 at 15 00 56" src="https://user-images.githubusercontent.com/32431609/161983052-a47cc2cc-c205-4337-9eca-46b9f81400f2.png">
